_drafts/agile.md

```diff
@@ -15,96 +15,96 @@ keywords:
   - agile is evil
 ---
 
-[Agile Manifesto](http://www.agilemanifesto.org/) is nowadays a Bible of
-so many software teams. It contains
-[12 principles](http://agilemanifesto.org/principles.html), which are
-supposed to show us how software development must be organized.
-They were invented in 2001.
-I like them and agree with all of them. However, in practice, most software teams
-misunderstand them all. Here is my summary of what's going on and my
-own interpretation of each principle.
+Nowadays, [Agile Manifesto](http://www.agilemanifesto.org/) is a Bible of
+numerous software teams. It contains
+[12 principles](http://agilemanifesto.org/principles.html) which
+ show us how software development should be organized.
+These principles were invented in 2001.
+Generally, I like and agree with all of them. However, in practice, most software teams
+misunderstand them. Consequently, here is a summary of what's going on and my
+interpretation of each principle.
 
 <!--more-->
 
 **Principle #1**:
 "Our highest priority is to satisfy the customer through early and continuous delivery of valuable software."
 
-Staying focused on "satisfy the customer" Agile adepts totally forget about
+By focusing on "satisfy the customer", Agile adepts totally forgets about
```

Comment:
No, you're right. Should be "Agile adepts forget". Though in this case,  I'm still rooting for "Agile experts..."